src/api/authorityApi.ts

```diff
@@ -93,7 +93,63 @@ export const createAuthority = async (user: User) => {
   }
 };
 
-export const removeIdFromAuthority = async (
+export const addQualifierIdForAuthority = async (
+  systemControlNumber: string,
+  qualifier: AuthorityQualifiers,
+  identifier: string
+) => {
+  const url = `${AuthorityApiPaths.AUTHORITY}/${systemControlNumber}/identifiers/${qualifier}/${identifier}`;
+
+  try {
+    const idToken = await getIdToken();
+    const headers = {
+      Authorization: `Bearer ${idToken}`,
+    };
+    const response = await Axios.post(url, { headers });
```

Comment:
all data sendes via URL og ingenting i body? hvis det er det backend vil så er det forsåvidt greit for min del, men det virker litt rart, eller?

Reply:
Ja, siden det bare er en id så sendes det sånn ja. Mulig det er litt rart, men når vi kun sender én id, så må det vel være enklere sånn enn å putte inn en body som kun inneholder id?

Prøvde å google litt, men fant ikke så mye. Fant denne da: https://stackoverflow.com/questions/4191593/is-it-considered-bad-practice-to-perform-http-post-without-entity-body

Men kanskje backend-teamet bør svare på denne. @brinxmat @nraanes

Reply:
Kan argumenteres med at dette er en update (= PATCH) på deler av authority og ikke en create (= POST)